<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>600</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>500</width>
         <height>500</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 161px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="title">
        <string>PMA algoritam - Input Form</string>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>390</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:pressed {
    background-color: rgba(23, 54, 93, 0.8);
}
QPushButton:hover {
    background-color: rgba(23, 54, 93, 0.6);
	cursor: pointer;
}</string>
        </property>
        <property name="text">
         <string>POKRENI PROCJENU MOBILNOSTI</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelForDisplay_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>411</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>421</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="category_wrapper">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Kategorija nestale osobe:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="category_dropdown">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>421</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Vremensko razdoblje nestanka:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEdit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>421</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Faktor uvjeta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ClosedHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
   border: 1px solid #FF17365D;
   height: 10px;
   background:  white;
}

QSlider::handle:horizontal {
    background: #FF17365D;
    border: 1px solid #FF17365D;
    width: 20px;
     margin: -8px 0px; 
	 border-radius: 5px;
}</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="labelForDisplay_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>421</width>
          <height>37</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Faktor predstavlja uvjete na ovom području. Vrijednost 1 označava najbolje uvjete, dok vrijednost 0 označava najgore uvjete.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>421</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="segment_size">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Površina segmenta u m2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="segment_size_dropdown">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
